lemur/destinations/service.py

```diff
@@ -41,19 +41,26 @@ def create(label, plugin_name, options, description=None):
     return database.create(destination)
 
 
-def update(destination_id, label, options, description):
+def update(destination_id, label, plugin_name, options, description):
```

Comment:
Function: Changing the plugin selected required updating the `plugin_name` field as well as the options.